app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class='container wrapper'>
+  <div class='row'>
+    <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
+      <%= form_for @recipe do |form| %>
+        <div class='form-group field-height'>
+          <label for='name'>Recipe name</label>
+          <%= form.text_field :name, class: 'form-control' %>
+          <span class='error-span'><%= @recipe.errors[:name].join(', ') %></span>
+        </div>
+        <div class='form-group field-height'>
+          <label for='unit type'>Instructions</label>
+          <%= form.text_field :instructions, class: 'form-control' %>
+          <span class='error-span'><%= @recipe.errors[:instructions].join(', ') %></span>
+        </div>
+        <%= form.fields_for :recipe_ingredients do |rec_ing| %>
+          <%= render "recipe_ingredient_fields", f: rec_ing %>
```

Comment:
Please try not to use single letter variables like `f`. You could name it `recipe_ingredient_fields`.

Reply:
I don't quite how does variable name impact how the gem will work. All you need to do is to just rename it here and in `app/views/recipes/_recipe_ingredient_fields.html.erb`.